pywinauto/unittests/test_application.py

```diff
@@ -566,6 +565,51 @@ def test_kill_(self):
         self.assertRaises(AttributeError, app.UntitledNotepad.Edit)
 
 
+    def test_connect_timeout(self):
+        """Test that connect_() works with a timeout"""
+        app1 = Application()
+        app1.start(_notepad_exe())
+
+        app_conn = Application()
+        if is_x64_Python() or not is_x64_OS():
+            app_conn.connect(path = r"c:\windows\system32\notepad.exe", timeout = 1)
+
+        else:
+            app_conn.connect(path = r"c:\windows\syswow64\notepad.exe", timeout = 1)
+
+        app_conn.UntitledNotepad.MenuSelect('File->Exit')
+
+
+    def test_connect_timeout_incorrect(self):
+        """Test that connect_() works with a timeout"""
+        app1 = Application()
+        app1.start(_notepad_exe())
+
+        app_conn = Application()
+        if is_x64_Python() or not is_x64_OS():
+            app_conn.connect(path = r"c:\windows\system32\notepad.exe", timeout = "1.0")
+
+        else:
+            app_conn.connect(path = r"c:\windows\syswow64\notepad.exe", timeout = "1.0")

```

Comment:
Yes, `notepad.exe` is a real world application that is not intended to be used everywhere in the unit tests.